extensions/reviewed/CrazyGamesAdApi.json

```diff
@@ -349,6 +349,7 @@
                 "    },",
                 "    adError: (error, errorData) => {",
                 "        gdjs._crazyGamesExtension.isPlayingVideoAd = false;",
+                "        gdjs._crazyGamesExtension.hasVideoAdJustFinishedPlaying = true;",
```

Comment:
Is it to ensure that games won't be stuck if they forgot to handle errors?
Should there be a way for creator to still check if it has finished because of an error?

I guess it's to comply to this:

> Often, the request for a midgame ad will be unfilled (either because of timing restrictions, adblock, or low demand). In this case you receive an adError event. You should handle this case correctly and ensure that the game continues.